entries in table theme_document are not mandatory

Entries in tables documents and theme_documents in the main schema are not mandatory although that will probably be the normal case. However, someone could start with cantonal themes only where all the documents are directly related to the PLR. This will cause an error.

I suggest to add an if statement to handle this case at https://github.com/openoereb/pyramid_oereb/blob/9d6d3138e4d70b1ffbed203cb1ca7120985bde43/pyramid_oereb/core/config.py#L133

```
.venv/lib/python3.7/site-packages/pyramid_oereb/core/config.py:70: in init
    Config.assemble_relation_themes_documents()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    @staticmethod
    def assemble_relation_themes_documents():
        for theme in Config.themes:
            theme_documents = []
>           for theme_document in Config.theme_document:
E           TypeError: 'NoneType' object is not iterable
```

--- a/pyramid_oereb/core/config.py
+++ b/pyramid_oereb/core/config.py
@@ -141,7 +141,7 @@ class Config(object):
                 if len(theme_documents) > 0:
                     theme.document_records = theme_documents
         else:
-            log.info('No global documents, related to themes were provided!')
+            log.info('No global documents related to themes were provided!')
 
     @staticmethod
     def _read_themes():

--- a/pyramid_oereb/core/config.py
+++ b/pyramid_oereb/core/config.py
@@ -128,17 +128,20 @@ class Config(object):
 
     @staticmethod
     def assemble_relation_themes_documents():
-        for theme in Config.themes:
-            theme_documents = []
-            for theme_document in Config.theme_document:
-                if theme_document.theme_id == theme.identifier:
-                    for document in Config.documents:
-                        if theme_document.document_id == document.identifier:
-                            document_copy = document.copy()
-                            document_copy.article_numbers = theme_document.article_numbers
-                            theme_documents.append(document_copy)
-            if len(theme_documents) > 0:
-                theme.document_records = theme_documents
+        if Config.theme_document is not None:
+            for theme in Config.themes:
+                theme_documents = []
+                for theme_document in Config.theme_document:
+                    if theme_document.theme_id == theme.identifier:
+                        for document in Config.documents:
+                            if theme_document.document_id == document.identifier:
+                                document_copy = document.copy()
+                                document_copy.article_numbers = theme_document.article_numbers
+                                theme_documents.append(document_copy)
+                if len(theme_documents) > 0:
+                    theme.document_records = theme_documents
+        else:
+            log.info('No global documents, related to themes were provided!')
 
     @staticmethod
     def _read_themes():

